src/main/java/com/alipay/oceanbase/hbase/OHTableClient.java

```diff
@@ -314,6 +339,13 @@ public boolean checkAndPut(byte[] row, byte[] family, byte[] qualifier, byte[] v
         return ohTable.checkAndPut(row, family, qualifier, value, put);
     }
 
+    @Override
+    public boolean checkAndPut(byte[] bytes, byte[] bytes1, byte[] bytes2,
+                               CompareFilter.CompareOp compareOp, byte[] bytes3, Put put)
+                                                                                         throws IOException {
+        return false;
```

Comment:
if not support, throw exception and add comment

Reply:
fix